src/main/webapp/app/view/create-project/project-create.component.ts

```diff
@@ -1,29 +1,28 @@
-import { Component, OnInit } from '@angular/core';
-import { HttpResponse } from '@angular/common/http';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { FormBuilder, Validators } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
-import { Observable } from 'rxjs';
-import { IProject } from 'app/shared/model/project.model';
+import { Subject } from 'rxjs';
 import { ProjectService } from 'app/entities/project/project.service';
-import { IUser } from '../../core/user/user.model';
-import { AccountService } from '../../core/auth/account.service';
-import { CreateProject, ICreateProject } from '../../shared/model/dto/create-project.model';
+import { IUser } from 'app/core/user/user.model';
+import { AccountService } from 'app/core/auth/account.service';
+import { CreateProject, ICreateProject } from 'app/shared/model/dto/create-project.model';
 
-import { AUTHORITY_ADMIN } from '../../shared/constants/authority.constants';
+import { AUTHORITY_ADMIN } from 'app/shared/constants/authority.constants';
 
 import * as moment from 'moment';
+import { switchMap, takeUntil } from 'rxjs/operators';
```

Comment:
` import * as moment from 'moment';` as last import, because it's a little special, please.